src/main/java/pageobject/LoginPage.java

```diff
@@ -0,0 +1,50 @@
+package pageobject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends AbstractPage {
+    private static final String URL = "https://network-international--qa.cs83.my.salesforce.com/";
+    private static final By USER_NAME_FIELD_LOCATOR = By.xpath("//input[@id='username']");
```

Comment:
use by.id for locators

Reply:
I used Xpath because to have more practice in it. And yes I know that search by id is faster.